app/controllers/protocols_controller.rb

```diff
@@ -105,6 +105,10 @@ def preview
     end
   end
 
+  def recent_protocols
+    render json: Protocol.recent_protocols(current_user, current_team, 14).select(:id, :name)
```

Comment:
Can you move that number to some constant?

Reply:
fix